examples/kepler-integration/package.json

```diff
@@ -11,6 +11,7 @@
     "global": "^4.3.0",
     "hubble.gl": "^1.1.0",
     "react-intl": "^3.12.0",
+    "react-map-gl": "^5.2.10",
```

Comment:
Oh on second look this would also move to the top level json devDep

Reply:
Added to root package.json

Reply:
Thanks. You'll also want to remove it here.